src/pymmcore_widgets/hcs/_fov_widget/_fov_sub_widgets.py

```diff
@@ -0,0 +1,768 @@
+        self._area_y.setValue(value.max_height)
+        self._fov_size = (value.fov_width, value.fov_height)
+
+    def reset(self) -> None:
+        """Reset the values of the widgets."""
+        self._number_of_points.setValue(1)
+        self._area_x.setValue(0)
+        self._area_y.setValue(0)
+        self._fov_size = (None, None)
+        self.is_circular = False
+
+
+class _GridFovWidget(QWidget):
+    """Widget to generate a grid of FOVs within a specified area."""
+
+    valueChanged = Signal(object)
+
+    def __init__(self, parent: QWidget | None = None) -> None:
+        super().__init__(parent)
+
+        self._fov_size: tuple[float | None, float | None] = (None, None)
+
+        self._rows = QSpinBox()
+        self._rows.setAlignment(AlignCenter)
+        self._rows.setMinimum(1)
+        rows_lbl = create_label("Rows:")
+        rows = make_wdg_with_label(rows_lbl, self._rows)
+        self._rows.valueChanged.connect(self._on_value_changed)
+
+        self._cols = QSpinBox()
+        self._cols.setAlignment(AlignCenter)
+        self._cols.setMinimum(1)
+        cols_lbl = create_label("Columns:")
+        cols = make_wdg_with_label(cols_lbl, self._cols)
+        self._cols.valueChanged.connect(self._on_value_changed)
+
+        self._overlap_x = QDoubleSpinBox()
+        self._overlap_x.setAlignment(AlignCenter)
+        self._overlap_x.setMinimum(-10000)
+        self._overlap_x.setMaximum(100)
+        self._overlap_x.setSingleStep(1.0)
+        self._overlap_x.setValue(0)
+        overlap_x_lbl = create_label("Overlap x (%):")
+        overlap_x = make_wdg_with_label(overlap_x_lbl, self._overlap_x)
+        self._overlap_x.valueChanged.connect(self._on_value_changed)
+
+        self._overlap_y = QDoubleSpinBox()
+        self._overlap_y.setAlignment(AlignCenter)
+        self._overlap_y.setMinimum(-10000)
+        self._overlap_y.setMaximum(100)
+        self._overlap_y.setSingleStep(1.0)
+        self._overlap_y.setValue(0)
+        spacing_y_lbl = create_label("Overlap y (%):")
+        overlap_y = make_wdg_with_label(spacing_y_lbl, self._overlap_y)
+        self._overlap_y.valueChanged.connect(self._on_value_changed)
+
+        self._order_combo = QComboBox()
+        self._order_combo.addItems([mode.value for mode in OrderMode])
+        self._order_combo.setCurrentText(OrderMode.row_wise_snake.value)
+        order_combo_lbl = create_label("Grid Order:")
+        order_combo = make_wdg_with_label(order_combo_lbl, self._order_combo)
+        self._order_combo.currentTextChanged.connect(self._on_value_changed)
+
+        # add widgets to wdg layout
+        self._wdg = QGroupBox()
+        layout = QVBoxLayout(self._wdg)
+        layout.setSpacing(5)
+        layout.setContentsMargins(10, 10, 10, 10)
+        title = QLabel(text="Fields of Views in a Grid.")
+        title.setStyleSheet("font-weight: bold;")
+        title.setAlignment(AlignCenter)
+        layout.addWidget(title)
```

Comment:
this layout, (and all of the `make_wdg_with_label` helpers) should be replaced with QFormLayout